src/BlazorWebView/samples/BlazorWpfApp/MainWindow.xaml.cs

```diff
@@ -18,12 +18,20 @@ public partial class MainWindow : Window
 		public MainWindow()
 		{
 			var services1 = new ServiceCollection();
-			services1.AddBlazorWebView();
+			services1.AddWpfBlazorWebView();
```

Comment:
Do we have error handling for the case when someone does `services1.AddWindowsFormsBlazorWebView();` in WPF?.

Edit: Looks like that's automatically handled by the ifdefs?

Reply:
> Do we have error handling for the case when someone does services1.AddWindowsFormsBlazorWebView(); in WPF?

There's not really a good way (if its even possible) to add error handling for this, but it's not going to be a problem because you would need to reference two separate packages.